source/blueconfig-projection-example.rst

```diff
@@ -37,42 +47,59 @@ How to properly add a Projection to a BlueConfig
         SynapseConfigure %s.Use = 0.86
     }
 
-How to generate a spikes file for Projection
+
+.dat Spike Files
+~~~~~~~~~~~~~~~~
+
+Spike files used with BlueConfigs follow a very simple ".dat" text spec.
+They have a `/scatter` header followed by lines of "<time-ms>  <cell-id>".
+
+Example:
+
+```
+/scatter
+015.7384     221086
+015.8529     222256
+015.8538     221131
+015.8726     221285
+```
+
+**SONATA COMPAT NOTE**: The <cell-id> is a 1-based NEURON cell Id, which is consistent
+with legacy MVD formats. In turn, SONATA uses 0-based cell Ids and therefore,
+in case you use SONATA node sets, you will need to OFFSET EVERY CELL ID BY 1.
+
+Building on the previous example, the first spike would go out at 15.7384ms on cell with
+SONATA Node Id 221085.
```

Comment:
Looks good to me. Just wondering whether the "spike_file" in `sonata_simulation.rst` can reference this note.